### PR TITLE
Add a document about how to implement a Mars operand

### LINKED ISSUE #2269: how to define a new operator or operand？
Note that the issue tracker is NOT the place for general support.  For
discussions about development, questions about usage, or any general questions,
we use our mailing list mars-dev@googlegroups.com.

### PR DESCRIPTION
**Pull request by hekaisheng**
## What do these changes do?

- [x] Add a document about how to implement a Mars operator in development guidelines.
- [x] Translation

## Related issue number

Fixes #2269

### CODE REVIEW

File: docs/source/development/operand.rst [hunk 1]
```diff
@@ -0,0 +1,253 @@
...
+
+    class SimpleReadCSV(DataFrameOperand, DataFrameOperandMixin):
+        path = StringField('path')
+
+        def __init__(self, path=None, **kw):
+            super().__init__(path=path, _output_types=[OutputType.dataframe], **kw,)
+
+        def __call__(self, index_value=None, columns_value=None,
+                     dtypes=None, chunk_bytes=None):
+            shape = (np.nan, len(dtypes))
```
Comment (qinxuye) [inline docs/source/development/operand.rst:33]: Add a doc to explain np.nan.

File: docs/source/development/operand.rst [hunk 2]
```diff
@@ -0,0 +1,253 @@
...
+        return op(
+            index_value=index_value,
+            columns_value=columns_value,
+            dtypes=dtypes,
+            chunk_bytes=chunk_bytes,
+        )
+
+Functional testing
+-------------------
+Write a script to test if the ``read_csv`` works.
```
Comment (qinxuye) [inline docs/source/development/operand.rst:223]: Tell the user how to write unittests.

**Review by qinxuye — COMMENTED** [review]
LGTM overall.

**Comment by qinxuye** [comment]
This PR explains how to implement an operand, from the developer perspective. What is your opinion, is that clear to you? @perfumescent

**Review by qinxuye — APPROVED** [review]
LGTM

**Review by wjsi — APPROVED** [review]
LGTM

**Pull request merged**